src/adaptors/http1/http1_client.c

```diff
@@ -815,6 +852,9 @@ static int _client_rx_body_cb(h1_codec_request_state_t *hrs, qd_buffer_list_t *b
 {
     _client_request_t       *hreq = (_client_request_t*) h1_codec_request_state_get_context(hrs);
     qdr_http1_connection_t *hconn = hreq->base.hconn;
+    if (hconn->cfg.event_channel && strcasecmp(h1_codec_request_state_method(hrs), POST_METHOD) != 0) {
+        return 0;
```

Comment:
Need to free the body buffer list with qd_buffer_list_free_buffers as ownership is assumed.

Reply:
done